vsphere/resource_vsphere_license.go

```diff
@@ -0,0 +1,287 @@
+package vsphere
+
+import (
+	"errors"
+	"fmt"
+	"log"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	"github.com/vmware/govmomi"
+	"github.com/vmware/govmomi/license"
+	"github.com/vmware/govmomi/vim25/methods"
+	"github.com/vmware/govmomi/vim25/types"
+	"golang.org/x/net/context"
```

Comment:
Can this be changed to the standard library `context`? So just `context`, versus `golang.org/x/net/context`.